packages/auth-clients/src/main/VerificationClient.test.ts

```diff
@@ -16,7 +16,7 @@ describe('VerificationClient', () => {
   }
 
   beforeEach(() => {
-    fakeApi = nock(config.url)
+    fakeApi = nock(config.url, { badheaders: ['Content-Type'] })
```

Comment:
We should never be setting Content-Type when verifying a token as we're not currently sending anything